File: 1128-remove-all-adjacent-duplicates-in-string/remove-all-adjacent-duplicates-in-string.py
class Solution:
    def removeDuplicates(self, s: str) -> str:
        output = []
        for i in range(0,len(s)):
            if output and output[-1] == s[i]:
                output.pop()
            else:
                output.append(s[i])
        return "".join(output)
        

File: 1128-remove-all-adjacent-duplicates-in-string/remove-all-adjacent-duplicates-in-string.js
/**
 * @param {string} s
 * @return {string}
 */
var removeDuplicates = function(s) {
    let mystack = [];
    let top = "";
    for(let i = 0;i<s.length;i++){
        if (top === s[i])
            mystack.pop();
        else
            mystack.push(s[i]);

        top = mystack[mystack.length-1];
    }
    return mystack.join("");
};